hands_on/lecture1_experimental_design.md

```diff
@@ -0,0 +1,42 @@
+# Hands-On Activity #1: Designing A Simple Computational Biology Experiment
+
+## Objective:
+
+Design a simple computational biology experiment. I encourage you to think about data and research questions from your current research project.
```

Comment:
For markdown convention we use 1-sentence-per-line. (I _think_ `pre-commit` will check for this, but I'm not sure, so considering running)

```suggestion
Design a simple computational biology experiment.
I encourage you to think about data and research questions from your current research project.
```